pillar/redash.sls

```diff
@@ -1,3 +1,17 @@
+network:
+  host_id: ocp14
+  ipv4: 139.162.199.85
+  ipv6: 2a01:7e00:e000:02cc::14
+  netplan:
+    configuration: linode
+    ipv4_subnet_mask: "/24"
+    addresses:
+      - 2a01:7e00::f03c:92ff:fea5:0e5f/128  # SLAAC
```

Comment:
Why is /128 added to the SLAAC IP? I only see the IP (not the mask) on Linode.

Reply:
I think I assumed it was a single IP address which should be indicated with `/128`.
Netplan requires a subnet_mask to be defined.

Looking again I see the Linode Networking page shows a subnet mask of `ffff:ffff:ffff:ffff::` which is `/64`.
This is also in the Linode documentation: https://www.linode.com/docs/guides/linux-static-ip-configuration/#ubuntu-1804-and-2004

Both have worked in my testing - we are not using SLAAC since we are using static configuration but we have to set it up for netplan.

Reply:
I'll update this to /64.